src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Random must be a class variable
```suggestion
        int index = random.nextInt(Color.values().length);
```